Notify/Notify/Notify/AppShell.xaml.cs

```diff
@@ -13,19 +16,24 @@
 using ProfilePage = Notify.Views.ProfilePage;
 using TeamDetailsPage = Notify.Views.TeamDetailsPage;
 
+
 namespace Notify
 {
+   
     [XamlCompilation(XamlCompilationOptions.Compile)]
     public partial class AppShell : Shell
     {
         private readonly INotificationManager notificationManager = DependencyService.Get<INotificationManager>();
         private Location m_LastUpdatedLocation = null;
-        
+        private readonly string my_ssid = "\"AndroidWifi\"";    // DEKEL
```

Comment:
Remove the blank space between class members.
What does the "\" suppose to mean in `"\"AndroidWifi\"`?
Consider changing the name of the member.
Remove the comment

Reply:
The SSID value returned by wifiManager.ConnectionInfo.SSID includes quotation marks.